src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, Dispatch, SetStateAction } from 'react'
+import * as FaIcons from 'react-icons/fa'
+import * as AiIcons from 'react-icons/ai'
+import styles from './navbar.module.scss'
+import { AppRoutes, Route, appRoutes } from '../../views/router/routes'
+import classNames from 'classnames'
```

Comment:
Will also need to clean up these imports (maybe else where in the PR, as well.)

Make sure you have ESLint enabled, as it will catch most of these for you.